05/index.html

```diff
@@ -33,15 +27,15 @@
 
     <main class="container__green main__section">
       <article class="container__blue container__main">
-        <div class="rectangle__orange rectangle__1"></div>
-        <div class="rectangle__orange rectangle__2"></div>
+        <div class="rectangle__orange rectangle__first-main"></div>
```

Comment:
Już nie zmieniaj tego, ale zmiana z `1` na `first`, nie wiele zmienia. Jeśli będziemy chcieli lub musieli zmienić ich kolejność to nazwa klasy będzie wprowadzać naw w błąd. Docelowo lepiej opisywać co zawiera ten element np. opis członków zespół, listą pakietów itp. Wiem, że w tym zadaniu trudno powiedzieć co on zawiera i stąd wynika numeracja ;)